packages/components/src/Tooltip/_docs/Tooltip.stories.tsx

```diff
@@ -12,6 +21,16 @@ const meta = {
   args: {
     text: "Example tooltip text.",
   },
+  decorators: [
+    Story => (
+      <div className="flex mt-[60px] gap-12">
+        <Story />
+      </div>
+    ),
+  ],
```

Comment:
you can remove the decorator now when you've set layout

Reply:
We still need the decorator to add some whitespace of each story so the Tooltip floats to the top instead of the sides. Could probably have the flex and gap specific to the button story but figured there was no harm since it won't impact any of the other stories that have only one element